omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,256 @@
+                                        <obs id="district-of-parent-health-facility" labelText="District of Parent Health Facility" conceptId="166454" />
+                                        <label>
+                                            <span class="required">*</span>
+                                            Serial No
+                                        </label>
+                                        <obs id="serial-number" conceptId="1646" />
+                                        <label>Encounter Date</label>
+                                        <encounterDate default="today" disallowMultipleEncountersOnDate="block" />
+                                    </div>
+
+                                    <div class="row">
+                                        <div class="col-md-12">
+                                            <obs id="reason-for-visit-other" labelText="Other Specify" conceptId="164975" />
+                                            <obs id="other-regimen" labelText="Other Regimen" conceptId="99126" />
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-3">
+                                        <label>
+                                            <span class="required">*</span>
+                                            Telephone No
+                                        </label>
+                                        <obs id="telephone-number" conceptId="90266" />
+                                        <label>
+                                            <span class="required">*</span>
+                                            PT Clinic No
+                                        </label>
+                                        <obs id="pt-clinic-number" conceptId="90264" />
+
+                                        <obs id="eid-number" labelText="EID Number" conceptId="166453" />
+
+                                        <obs id="client-category" conceptId="165169" labelText="Client Category" answerConceptIds="90315, 165132, 159674, 162198, 165125, 90041, 99601, 157351, 99087,165133, 159991, 165127, 166461, 166462, 166463" answerLabels="MW- Migrant Workers, AGYW- Adoloscent Girls and Young Women, FF- Fisher Folk, LDT- Long Distance Truck Drivers, UF- Uniformed Forces, Preg- Pregnant, BF- Breastfeeding, IDU- Injectable Drug User, HEI- HIV Exposed Infant, OKP- Other KPs, TB- TB/HIV Client, RF- Refugee, ART- General ART Client, NIDU- Non IDU, OPP- Other PPs" />
+                                        <label>
+                                                    Nin Number
```

Comment:
@ODORA0 Change this to National ID Number

Reply:
@ssmusoke Done